Tests/SentryTests/SentryScopeSwiftTests.swift

```diff
@@ -231,17 +231,6 @@ class SentryScopeSwiftTests: XCTestCase {
         XCTAssertEqual(trace?["span_id"] as? String, fixture.transaction.spanId.sentrySpanIdString)
     }
     
-    func testApplyToEvent_ScopeWithSpan_NotAppliedToCrashEvent() {
```

Comment:
I moved this test to the client tests `testCaptureCrashEventWithSession_ScopeWithSpan_NotAppliedToCrashEvent`

Reply:
Thanks for mentioning.